js-packages/@prestojs/viewmodel/src/fields/DateField.ts

```diff
@@ -21,6 +21,19 @@ export default class DateField extends Field<Date, string | Date> {
         return new Date(value);
     }
 
+    isEqual(a?: Date, b?: Date): boolean {
+        if (!a || !b) {
+            return false;
```

Comment:
should it be considered equal if both values are none?

Reply:
🤦‍♂️  thank you - definitely